spec.html

```diff
@@ -1585,158 +1585,121 @@ <h1>Number::bitwiseNOT ( _x_ )</h1>
-            <li>If _base_ is *-0*<sub>𝔽</sub> and ℝ(_exponent_) &gt; 0 and _exponent_ is an odd integral Number, the result is *-0*<sub>𝔽</sub>.</li>
-            <li>If _base_ is *-0*<sub>𝔽</sub> and ℝ(_exponent_) &gt; 0 and _exponent_ is not an odd integral Number, the result is *+0*<sub>𝔽</sub>.</li>
-            <li>If _base_ is *-0*<sub>𝔽</sub> and ℝ(_exponent_) &lt; 0 and _exponent_ is an odd integral Number, the result is *-&infin;*<sub>𝔽</sub>.</li>
-            <li>If _base_ is *-0*<sub>𝔽</sub> and ℝ(_exponent_) &lt; 0 and _exponent_ is not an odd integral Number, the result is *+&infin;*<sub>𝔽</sub>.</li>
-            <li>If ℝ(_base_) &lt; 0 and _base_ is finite and _exponent_ is finite and _exponent_ is not an integral Number, the result is *NaN*.</li>
-          </ul>
+          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-approximated value representing the result of raising _base_ to the _exponent_ power. It performs the following steps when called:</p>
+          <emu-alg>
+            1. If _exponent_ is *NaN*, return *NaN*.
+            1. If _exponent_ is *+0*<sub>𝔽</sub> or _exponent_ is *-0*<sub>𝔽</sub>, return *1*<sub>𝔽</sub>.
+            1. If _base_ is *NaN*, return *NaN*.
+            1. If _base_ is *+&infin;*<sub>𝔽</sub>, then
+              1. If _exponent_ &gt; *+0*<sub>𝔽</sub>, return *+&infin;*<sub>𝔽</sub>. Otherwise, return *+0*<sub>𝔽</sub>.
+            1. If _base_ is *-&infin;*<sub>𝔽</sub>, then
+              1. If _exponent_ &gt; *+0*<sub>𝔽</sub>, then
+                1. If _exponent_ is an odd integral Number, return *-&infin;*<sub>𝔽</sub>. Otherwise, return *+&infin;*<sub>𝔽</sub>.
+              1. Else,
+                1. If _exponent_ is an odd integral Number, return *-0*<sub>𝔽</sub>. Otherwise, return *+0*<sub>𝔽</sub>.
+            1. If _base_ is *+0*<sub>𝔽</sub>, then
+              1. If _exponent_ &gt; *+0*<sub>𝔽</sub>, return *+0*<sub>𝔽</sub>. Otherwise, return *+&infin;*<sub>𝔽</sub>.
+            1. If _base_ is *-0*<sub>𝔽</sub>, then
+              1. If _exponent_ &gt; *+0*<sub>𝔽</sub>, then
+                1. If _exponent_ is an odd integral Number, return *-0*<sub>𝔽</sub>. Otherwise, return *+0*<sub>𝔽</sub>.
+              1. Else,
+                1. If _exponent_ is an odd integral Number, return *-&infin;*<sub>𝔽</sub>. Otherwise, return *+&infin;*<sub>𝔽</sub>.
+            1. Assert: _base_ is finite and is neither *+0*<sub>𝔽</sub> nor *-0*<sub>𝔽</sub>.
+            1. If _exponent_ is *+&infin;*<sub>𝔽</sub>, then
+              1. If abs(ℝ(_base_)) &lt; 1, return *+0*<sub>𝔽</sub>.
+              1. If abs(ℝ(_base_)) is 1, return *NaN*.
+              1. If abs(ℝ(_base_)) &gt; 1, return *+&infin;*<sub>𝔽</sub>.
+            1. If _exponent_ is *-&infin;*<sub>𝔽</sub>, then
+              1. If abs(ℝ(_base_)) &lt; 1, return *+&infin;*<sub>𝔽</sub>.
```

Comment:
Nit: I would prefer we stick to consistently treating cases like this in descending order: so `> 1`, `= 1`, `< 1`, rather than the reverse.